enamel/main.py

```diff
@@ -66,10 +79,13 @@ def home():
 
 
 def server_boot():
+    if flask.request.content_type != 'application/json':
+        flask.abort(400)
```

Comment:
Also, not for now, but eventually this is probably something we're going to want for most methods so we may want to find a decorator or something that enforces this. I would guess that something is already in the flask stack the does it.